beetsplug/fetchart.py

```diff
@@ -330,19 +356,12 @@ def get(self, album, plugin, paths):
             size_suffix = "-" + str(plugin.maxwidth)
 
         if 'release' in self.match_by and album.mb_albumid:
-            if size_suffix:
-                release_thumbnail_url = release_url + size_suffix
-                yield self._candidate(url=release_thumbnail_url,
-                                      match=Candidate.MATCH_EXACT)
-            yield self._candidate(url=release_url,
-                                  match=Candidate.MATCH_EXACT)
+            url = get_image_url(release_url, size_suffix)
+            yield self._candidate(url=url, match=Candidate.MATCH_EXACT)
+
```

Comment:
According to the comment above, this should probably be
```Python
for url in get_image_urls(release_url, size_suffix):
    yield self._candidate(url=url, match=Candidate.MATCH_EXACT)
```

Reply:
You are 100% correct. No idea how that slipped through. Fixed.